components/next_steps_view/next_steps_view.scss

```diff
@@ -0,0 +1,88 @@
+#app-content.NextStepsView {
+    background-image: url(../../images/onboarding-bg.svg);
+    background-repeat: no-repeat;
+    background-position-y: bottom;
+    background-position-x: -305px;
+    background-size: calc(100% + 305px);
+}
+
+.NextStepsView__header {
+    padding: 32px 40px;
+    display: flex;
+}
+
+.NextStepsView__header-headerText {
+
+}
+
+.NextStepsView__header-headerTopText {
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 40px;
+    letter-spacing: -0.02em;
+}
+
+.NextStepsView__header-headerBottomText {
+    font-size: 16px;
+    line-height: 24px;
+    margin-top: 8px;
+}
+
+.NextStepsView__header-logo {
+    margin-left: auto;
+    align-self: center;
+}
+
+.NextStepsView__body {
+    display: flex;
+    height: 100%;
+}
```

Comment:
Have you thought about how you are going to work in the animated transitions between the various stage groups? Like when the graphic of the computer animates to the centre and changes the screen contents to act as a loading screen before switching to the Tips and Next Steps?

Reply:
I've put more thought into it in this PR: https://github.com/mattermost/mattermost-webapp/pull/5893
But I don't know for sure how that's supposed to work, was going to follow up with Matt when necessary.
My assumption was that they would be completely different `div`'s that I could switch around with transitions as needed.